modules/nf-core/sourmash/sketch/meta.yml

```
@@ -20,8 +20,8 @@ input:
         e.g. [ id:'test', single_end:false ]
   - sequence:
       type: file
-      description: FastA file containing (genomic) sequence data
-      pattern: "*.{fna,fa,fasta}"
+      description: FASTA or FASTQ file containing (genomic, transcriptomic, or protein) sequence data
+      pattern: "*.{fna,fa,fasta,fastq,fq}"
```

should I add `*gz` to these file format endings as well?

---

Yes, absolutely, nf-core actually prefers compressed file formats so maybe drop the uncompressed ones. Should also add `.faa` for amino acid sequences then?

---

so maybe something like this?
```suggestion
      pattern: "*.{fna,fa,fasta,fastq,fq,faa}.gz"
```

---

Neat, yes 🙂